data/emulator.js

```diff
@@ -452,16 +452,18 @@ class EmulatorJS {
         this.elements.parent.appendChild(this.textElem);
     }
     localization(text, log) {
-        if (!isNaN(text)) return text;
-        if (text.includes("EmulatorJS v")) return text;
-        if (this.config.langJson) {
-            if (typeof log === "undefined") log = true;
-            if (!this.config.langJson[text] && log) {
-                if(!window.EJS_missingLang) window.EJS_missingLang = [];
-                window.EJS_missingLang.push(text);
-                console.log("Translation not found for '"+text+"'. Language set to '"+this.config.language+"'");
+        if (typeof text !== "undefined"){
+            text = text.toString();
+            if (text.includes("EmulatorJS v")) return text;
+            if (this.config.langJson) {
+                if (typeof log === "undefined") log = true;
+                if (!this.config.langJson[text] && log) {
+                    if(!window.EJS_missingLang) window.EJS_missingLang = [];
```

Comment:
(there should be a space after the if and before the `(`)

Can we define these within the emulatorjs instance?
`this.missingLanguages = []`